### ISSUE TITLE
requirePaddingNewLinesAfterBlocks and object literal after function argument

### ISSUE THREAD

**Issue by canac** [jscs-dev/node-jscs#1798]
When checking the following code with the `requirePaddingNewLinesAfterBlocks` rule enabled, JSCS version 2.1.1 wants a newline between lines 3 and 4, between the object literal's opening brace and its first property:

```javascript
func(function(response) {
  console.log(response);
}, {
  a: 1,
  b: 2,
  c: 3,
});
```

Having an object literal argument after an anonymous function argument is a rare pattern, but the current behavior is still undesirable.

**Comment by markelog** [comment]
Did you try `inCallExpressions` value?

**Comment by canac** [comment]
@markelog You are right, when using `inCallExpressions`, JSCS no longer emits any warnings. My original concern was that there is no reasonable way to the format code in the example I provided. Then I realized that I should really be adding another level of indentation, like this:
```javascript
func(
  function(response) {
    console.log(response);
  },

  {
    a: 1,
    b: 2,
    c: 3,
  }
);
```

Instead of this:
```javascript
func(function(response) {
  console.log(response);
}, {
  a: 1,
  b: 2,
  c: 3,
});
```

Since I shouldn't really be writing code using the latter format, it makes sense that JSCS doesn't really support that format. Thanks for your help and for making such a useful tool!
